src/Field/ContentBuilder.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Message\Cog\Field;
+
+class ContentBuilder
```

Comment:
Please add docblocks, I know this is copy/pasted so don't worry too much about description but it would be nice to have the  param/return tags